lib/VisualGenerator.js

```diff
@@ -240,7 +248,7 @@ class VisualGenerator {
 
                     copyVisualTemplate(visualPath, buildOptions.template);
                     generateSettings(visualPath);
-                    createPbiVizJson(visualPath, visualOptions);
+                    createPbiVizJson(visualPath, visualOptions, buildOptions.externalJS);
```

Comment:
like before - let's not add buildOptions.externalJS but get it working with pbiviz.json per template